### DISCUSSION TITLE
Example of basic usage with typescript for V5

### DISCUSSION THREAD

**Discussion by atissedredecathlon** [JedWatson/react-select#4978] [Q&A] [answered]
I have trouble finding a basic implementation example of the select with typescript in V5.

I have several errors / warning about types, and I am not able to find anything in the current documentation.

Thank you for your help.

**Comment by Methuselah96** [comment, marked as answer]
All of the source code examples on the website are written in TypeScript and there is [a page](https://react-select.com/typescript) dedicated to explaining some of the details of the Select generics.

**Reply by atissedredecathlon** [reply]
Thank you, I didn't even see that the source code was visible on the examples.
